evm/src/all_stark.rs

```diff
@@ -271,45 +271,18 @@ mod tests {
             cpu_trace_rows.push(row.into());
         }
 
-        for i in 0..num_logic_rows {
+        // Pad to `num_memory_ops` for memory testing.
```

Comment:
The memory test and logic/syscall/jump tests now both use the memory value columns. Give the memory test its own rows to avoid conflicts (previously it modified existing rows).